### PR TITLE
#1628  Refactor sort table component making it more reusable

### PR DESCRIPTION
**Pull request by wu87988622**
### Checklist
- [x] [Review Contribution Guidelines](https://github.com/oharastream/ohara/blob/master/CONTRIBUTING.md)
- [x] Add Reviewers (see [Authors](https://github.com/oharastream/ohara#authors))
- [x] Add Assignees (of course, you are the assignee by default!)
- [x] DON'T set a Milestone
- [x] DON'T add any labels (PR labels will be added automatically by Jenkins)
- [x] Link to an issue(You can do so by using the required by keyword: required by #1628 )
- [ ] Add unit tests (or please explain why this PR is not worth having new tests)
- [ ] Pass QA (Sometimes you may encounter an existing flaky test which would obstruct you from getting QA passed. If the failed tests are unrelated, you can add a comment in the PR thread)

### CODE REVIEW

**Comment by wu87988622** [comment]
retry manager-it

**Comment by wu87988622** [comment]
retry manager-it

File: ohara-manager/client/src/components/pages/WorkspacesPage/WorkspacesDetailPage/__tests__/Topics.test.js [hunk 1]
```diff
@@ -61,8 +61,8 @@ describe('<Topics />', () => {
 
     const topicName = getByTestId('topic-name').textContent;
     const partitions = Number(getByTestId('topic-partitions').textContent);
-    const replications = Number(getByTestId('topic-replications').textContent);
-    const lastModified = getByTestId('topic-last-modified').textContent;
+    const replications = Number(getByTestId('topic-replication').textContent);
```
Comment (konekoya) [inline ohara-manager/client/src/components/pages/WorkspacesPage/WorkspacesDetailPage/__tests__/Topics.test.js]: Why removing the `s`...

File: ohara-manager/client/src/components/pages/WorkspacesPage/WorkspacesDetailPage/__tests__/Topics.test.js [hunk 2]
```diff
@@ -61,8 +61,8 @@ describe('<Topics />', () => {
...
+    const lastModified = getByTestId('topic-lastModified').textContent;
```
Comment (konekoya) [inline ohara-manager/client/src/components/pages/WorkspacesPage/WorkspacesDetailPage/__tests__/Topics.test.js]: We're not using camelcase in id either

**Pull request merged**